docs/INSTALL.md

```diff
@@ -1,17 +1,28 @@
 # skyWM Installation Guide
 
-* While this installation guide will work cross-distribution, we have only documented the dependencies for Ubuntu and Ubuntu-based systems for now. Please feel free to contribute by creating a pull request.
+* Please make a pr if any of these install scripts don't work
```

Comment:
Please can you edit this to say: "While this installation guide will work cross-distribution, we have only documented the dependencies for a few distributions at the moment. Feel free to contribute by creating a Pull Request"